kafka-hello-world/src/main/scala/com/labs1904/hwe/homework/KafkaHomework.scala

```diff
@@ -0,0 +1,65 @@
+package com.labs1904.hwe.homework
+
+import java.time.Duration
+import java.util.{Arrays, Properties, UUID}
+
+import net.liftweb.json.DefaultFormats
+import org.apache.kafka.clients.consumer.{ConsumerConfig, ConsumerRecord, ConsumerRecords, KafkaConsumer}
+import org.apache.kafka.common.serialization.StringDeserializer
+
+object KafkaHomework {
```

Comment:
Do we want to tie this to theory.md somehow? Right now they are two distinct entities

Reply:
Yea I like that, do you have any ideas on how to tie it in? Ideally we could do it in a way that wasn't too complicated